index.js

```diff
@@ -83,7 +84,9 @@ function exit() {
 		stack(results);
 	}
 
-	process.exit(stats.failCount > 0 ? 1 : 0);
+	if (process.exit) {
```

Comment:
How will this work with `hihat`. According to its docs:

> The process will stay open until you call window.close()

Wouldn't it be better to call `window.close` here if in the `browser`. Or even better, keep it as before, and have `hihat` shim `process.exit` => https://github.com/Jam3/hihat/issues/27